contracts/lib/time_series/running_stats.cairo

```diff
@@ -0,0 +1,65 @@
+%lang starknet
+
+from starkware.cairo.common.bool import TRUE, FALSE
+from starkware.cairo.common.math import unsigned_div_rem, abs_value, sqrt
+from starkware.cairo.common.math_cmp import is_le, is_nn
+
+from time_series.utils import is_positive, greater_than, safe_div
+
+# https://www.johndcook.com/blog/standard_deviation/
+# https://onlinegdb.com/V6FwrXOKA
+
+struct RunningStats:
+    member m_n : felt
+    member m_oldM : felt
+    member m_newM : felt
+    member m_oldS : felt
+    member m_newS : felt
```

Comment:
would prefer clearer names, even if longer

Reply:
this is a port of `RunningStat.h`, which is a C utility.  These are just mathematical constants.  You can read more here: https://www.johndcook.com/blog/standard_deviation/

https://github.com/tjhladish/AbcSmc/blob/master/RunningStat.h